src/main/java/com/lalabrand/ecommerce/auth/RefreshTokenService.java

```diff
@@ -0,0 +1,49 @@
+package com.lalabrand.ecommerce.auth;
+
+import com.lalabrand.ecommerce.user.UserRepository;
+import jakarta.transaction.Transactional;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.time.Instant;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class RefreshTokenService {
+    private final RefreshTokenRepository refreshTokenRepository;
+    private final UserRepository userRepository;
+    @Value("${refresh.token.expiration.seconds}")
+    private Long refreshTokenExpirationSec;
+
+    public RefreshTokenService(RefreshTokenRepository refreshTokenRepository, UserRepository userRepository) {
+        this.refreshTokenRepository = refreshTokenRepository;
+        this.userRepository = userRepository;
+    }
+
+    @Transactional
+    public RefreshToken createRefreshToken(String email) {
```

Comment:
@denomelchenko Would it be better to be able to create refresh token without fetching the user from the `userRepository`? You can pass the user object as a parameter.